pytorch_lightning/core/lightning.py

```diff
@@ -146,6 +149,78 @@ def forward(self, x):
         if self.trainer.is_global_zero:
             print(*args, **kwargs)
 
+    def log(
+        self,
+        name: str,
+        value: Any,
+        prog_bar: bool = False,
+        logger: bool = True,
+        on_step: bool = True,
+        on_epoch: bool = True,
+        reduce_fx: Callable = torch.mean,
+        tbptt_reduce_fx: Callable = torch.mean,
+        tbptt_pad_token: int = 0,
+        enable_graph: bool = False,
+        sync_dist: bool = False,
+        sync_dist_op: Union[Any, str] = 'mean',
+        sync_dist_group: Optional[Any] = None,
+    ):
+        """
+        Log a key, value
+
+        Example::
+
+            result.log('train_loss', loss)
+
+            # defaults used
+            result.log(
+                name,
+                value,
+                on_step=False,
+                on_epoch=False,
+                logger=True,
+                prog_bar=False,
+                reduce_fx=torch.mean,
+                enable_graph=False
+            )
+
```

Comment:
```suggestion
```
you do not like long docstring and this has zero information value

Reply:
? the docstrings are there... did you look?

Reply:
![image](https://user-images.githubusercontent.com/3640001/94374093-e3845000-00d7-11eb-8757-07437bfc02af.png)

Reply:
Yes and I recommend to remove the lines with default as it is already in function fingerprint